README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.5+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 I guess you can keep it like that, if you want. Xenia states that 2.7 is required and doesn't have + sign to indicate that versions above work. For Nucleus, in README I don't even see anything referring to the Python version required. Still you should change 3.5+ to 3.0+, because versions below 3.5 work, as should the 3.* versions starting from 3.0.